app/controllers/collections_controller.rb

```diff
@@ -53,6 +53,25 @@ def update
         end
       end
       @collection.update_attributes({:funder => newfunder})
+      @collection[:identifier].each do |id|
+        if /\Adoi:10.5072\/FK2/ =~ id
```

Comment:
@yinlinchen I don't think so. 'doi:10.xxxx' doesn't inherently mean anything, unless you know only DOIs we've minted start with that. It looks like a magic string to me.